### ISSUE TITLE
polaris standalone aarch64 install.sh error

### ISSUE THREAD

**Issue by xywf221** [polarismesh/polaris#1090]
**Describe the bug**
单利模式在linux aarch64 即 arm64 安装会报错

```
machine arch is aarch64, but Installation package arch is arm64
```

把arch.txt改成aarch64就行了

**Comment by chuntaojun** [comment]
will fix in release-v1.17.0

**Comment by i77yomi** [comment]
> will fix in release-v1.17.0
with polaris-standalone-release_v1.17.5.linux.arm64 ,I encountered another error:
machine arch is arm64, but Installation package arch is arm64

**Comment by i77yomi** [comment]
with polaris-standalone-release_v1.17.5.linux.arm64 ,I encountered another error:
machine arch is arm64, but Installation package arch is arm64
